pkg/storage/stores/index/labelvolume/labelvolume.go

```diff
@@ -0,0 +1,101 @@
+package labelvolume
```

Comment:
Interesting infrastructure to enable accumulation of label volumes. Of particular interest is how they're merged.